api/app/wildfire_one/schema_parsers.py

```diff
@@ -128,35 +130,26 @@ def generate_station_daily(raw_daily, station: WFWXWeatherStation, fuel_type: st
     """
     # pylint: disable=invalid-name
     # we use the fuel type lookup to get default values.
-    pc = FUEL_TYPE_LOOKUP[fuel_type]["PC"]
-    pdf = FUEL_TYPE_LOOKUP[fuel_type]["PDF"]
-    cbh = FUEL_TYPE_LOOKUP[fuel_type]["CBH"]
-    cfl = FUEL_TYPE_LOOKUP[fuel_type]["CFL"]
+    pc = FUEL_TYPE_DEFAULTS[fuel_type]["PC"]
+    pdf = FUEL_TYPE_DEFAULTS[fuel_type]["PDF"]
+    cbh = FUEL_TYPE_DEFAULTS[fuel_type]["CBH"]
+    cfl = FUEL_TYPE_DEFAULTS[fuel_type]["CFL"]
 
     isi = raw_daily.get('initialSpreadIndex', None)
     bui = raw_daily.get('buildUpIndex', None)
     ffmc = raw_daily.get('fineFuelMoistureCode', None)
-    fmc = cffdrs.foliar_moisture_content(station.lat, station.long, station.elevation, get_julian_date_now())
-
-    sfc = None
-    try:
-        sfc = cffdrs.surface_fuel_consumption(fuel_type, bui, ffmc, pc)
-    except cffdrs.CFFDRSException as exception:
-        logger.error(exception, exc_info=True)
+    fmc = cffdrs.foliar_moisture_content(
+        station.lat, station.long, station.elevation, get_julian_date_now())
+    sfc = cffdrs.surface_fuel_consumption(
+        FuelTypeEnum[fuel_type], bui, ffmc, pc)
 
     cc = raw_daily.get('grasslandCuring')
     if cc is None:
-        cc = FUEL_TYPE_LOOKUP[fuel_type]["CC"]
-
-    ros = None
-    try:
```

Comment:
Similar to above, `sfc` may be `None` if we don't have all required parameters from WFWX, which will break when calculated `ros`

Reply:
same as above - `rate_of_spread()` in `cffdrs` performs a check on whether any of the necessary params are `None`, and raises a `CFFDRSException` there if so.